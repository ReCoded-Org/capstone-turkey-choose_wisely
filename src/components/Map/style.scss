.map {
    width: 699px;
    padding: 2rem 0;
}