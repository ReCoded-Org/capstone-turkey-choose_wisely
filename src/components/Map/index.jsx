import React from "react";
import { Container } from "react-bootstrap";
import uniMap from "./../../images/uni.jpg";
import "./style.scss";

function Map() {
  return (
    <Container className="text-center">
      <img src={uniMap} className="map" alt="Map of Universities in Turkey" />
    </Container>
  );
}

export default Map;
