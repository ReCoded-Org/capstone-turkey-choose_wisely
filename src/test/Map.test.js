import React from "react";
import Map from "./../components/Map/index.jsx";
import renderer from "react-test-renderer";

test("Snapshop test of Map component", () => {
  const map = renderer.create(<Map />);
  let tree = map.toJSON();

  expect(tree).toMatchSnapshot();
});
