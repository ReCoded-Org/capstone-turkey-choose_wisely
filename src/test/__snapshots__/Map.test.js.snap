// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop test of Map component 1`] = `
<div
  className="text-center container"
>
  <img
    alt="Map of Universities in Turkey"
    className="map"
    src="uni.jpg"
  />
</div>
`;
